fadingpromises/schinese/setup-win32.tra

```diff
@@ -0,0 +1,35 @@
+@0    = ~褪色的承诺 汉化：Archenemy~
```

Comment:
You need to keep only `@0` string in the installation file. All other translations should stay in regular setup.tra